hbase-protocol-shaded/src/main/protobuf/server/master/Master.proto

```diff
@@ -1225,6 +1225,12 @@ message ScheduleServerCrashProcedureResponse {
   repeated uint64 pid = 1;
 }
 
+message ScheduleServerCrashProceduresForUnknownServersRequest {}
+
+message ScheduleServerCrashProceduresForUnknownServersResponse {
```

Comment:
I wanted to use similar naming to the existing SCP but it is getting too long. I'm open for suggestions how it should be named.
`RecoverUnknownServers`?

Reply:
Maybe `ScheduleSCPsForUnknownServersRequest` and `ScheduleSCPsForUnknownServersResponse`?